labonneboite/web/user/views.py

```diff
@@ -74,7 +75,7 @@ def account_delete():
     return render_template('user/account_delete.html', **context)
 
 
-@userBlueprint.route('/favorites/list', methods=['GET'])
+@userBlueprint.route('/favorites/list')
```

Comment:
Par simple curiosité, il y a t-il un intérêt à ne plus limiter à la méthode GET

Reply:
Par défaut, si on ne précise pas `methods=...`, la vue est limitée à GET. C'est ce qu'on fait pour toutes les autres vues, sauf erreur de ma part.

Reply:
Je pensais que c'était ouvert à toutes les méthodes par défaut mais c'est noté du coup :+1: